packages/jest-core/src/cli/index.ts

```diff
@@ -167,13 +155,28 @@ const buildContextsAndHasteMaps = async (
   return {contexts, hasteMapInstances};
 };
 
-const _run10000 = async (
+/**
+ * Runs Jest either in watch mode or as a one-off. This is a lower-level API than `runCLI` and is intended for internal use by `runCLI` or externally.
+ * Note that `process.exit` might be called when using `globalConfig.watch` or `globalConfig.watchAll` is true.
+ *
+ * @param globalConfig The global configuration to use for this run. It can be obtained using `readConfigs` (imported from 'jest-config').
```

Comment:
Example below explains this better.

```suggestion
 * @param globalConfig The global configuration to use for this run.
```